applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -677,14 +693,29 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 	return 0;
 }
 
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay)
 {
+	if (bitrate == NULL && sampling_rate == NULL && pres_delay == NULL) {
+		LOG_ERR("No valid pointers received");
+		return -ENXIO;
+	}
+
 	/* Get the configuration for the sink stream */
 	int frames_per_sec = 1000000 / bt_codec_cfg_get_frame_duration_us(audio_streams[0].codec);
 	int bits_per_frame = bt_codec_cfg_get_octets_per_frame(audio_streams[0].codec) * 8;
 
-	*sampling_rate = bt_codec_cfg_get_freq(audio_streams[0].codec);
-	*bitrate = frames_per_sec * bits_per_frame;
+	if (sampling_rate != NULL) {
+		*sampling_rate = bt_codec_cfg_get_freq(audio_streams[0].codec);
```

Comment:
One last question: Should we check that audio_streams is not NULL in the start of this function (le_audio_config_get)?

Reply:
Added check